src/ApparentHorizons/SpherepackIterator.cpp

```diff
@@ -0,0 +1,160 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+//
+// SPHEREPACK memory layout
+//
+// Given a S2 with n_th x n_ph grid points, the spectral coefficients
+// are stored as follows:
+//
+//                            m_max+1
+//     -----------------------------------------------------
+//     | (0,0)         x       x            x
+//     | (1,0)       (1,1)     x            x         ARRAY
+//     | (2,0)       (2,1)   (2,2)          x          "a"
+//     |   :           :       :            x
+//     | (m_max,  0)  ...            (m_max,   m_max)
+//     | (m_max+1,0)  ...            (m_max+1, m_max)
+//     |    :                  :            :
+//     | (l_max, 0)                  (l_max,   m_max)
+//     -----------------------------------------------------
+//     |    x        x          x          x
+//     |    x      (1,1)        x          x          ARRAY
+//     |    x      (2,1)      (2,2)        x           "b"
+//     |    :        :          :          x
+//     |    x    (m_max,  1)..      (m_max,   m_max)
+//     |    x    (m_max+1,1)..      (m_max+1, m_max)
+//     |    :         :               :
+//     |    x    (l_max,  1)..      (l_max,   m_max)
+//     -----------------------------------------------------
+//
+// This layout has 2*n_th rows and m_max+1 columns,
+// the numbers across a *row* vary fastest.
+// 'x' stands for an entry that is not referenced by SPHEREPACK.
+// (Note that there is one complete unreferenced row in the middle)
+//
+// In the above:
+// - l_max is equal to n_th-1,
+// - m_max is equal to min(l_max,n_ph/2) [integer division]
+// So the full size of each array is (l_max+1)*(m_max+1)
```

Comment:
Remove this block (it defines useful quantities, so it makes more sense to put it before the table).